apps/api/src/app/modules/user/users.controller.ts

```diff
@@ -1,16 +1,25 @@
 import {
+  BadRequestException,
+  Body,
   Controller,
   Get,
+  HttpCode,
+  HttpStatus,
+  Post,
+  Put,
   UseGuards,
 } from '@nestjs/common';
 import { UserService } from './user.service';
 import { ApiTags } from '@nestjs/swagger';
 import { UserDTO } from './dto/user.dto';
 import { AuthGuard } from '@nestjs/passport';
 import { CurrentUser } from '../auth/decorator/current-user';
-import { IUser } from '@libs/types';
+import { ChangePasswordInput, IUser } from '@libs/types';
```

Comment:
Use `I` prefix like `IChangePasswordInput` for interface every where in api.

Reply:
done